Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1565,8 +1565,8 @@ RefPtr<Gfx::Font const> StyleComputer::find_matching_font_weight_descending(Vect
 // FIXME: This should be replaced by the full CSS font selection algorithm.
 RefPtr<Gfx::Font const> StyleComputer::font_matching_algorithm(FontFaceKey const& key, float font_size_in_pt) const
 {
-    // If a font family match occurs, the user agent assembles the set of font faces in that family and then
-    // narrows the set to a single face using other font properties in the order given below.
+    // 5.2.3 For other family names, the user agent attempts to find the family name among fonts defined via
```

Comment:
There's no need to prefix the step numbers with the section (5.2). Just the number of the step (3 here) is fine. Same for all the other numbered steps.

Reply:
Thank you for the feedback. I've removed the section prefixes.